src/inc/msquic.h

```diff
@@ -118,6 +118,7 @@ typedef enum QUIC_CREDENTIAL_FLAGS {
     QUIC_CREDENTIAL_FLAG_USE_SYSTEM_MAPPER                      = 0x00010000, // Schannel only
     QUIC_CREDENTIAL_FLAG_CACHE_ONLY_URL_RETRIEVAL               = 0x00020000, // Windows only currently
     QUIC_CREDENTIAL_FLAG_REVOCATION_CHECK_CACHE_ONLY            = 0x00040000, // Windows only currently
+    QUIC_CREDENTIAL_FLAG_INPROC_PEER_CERT                       = 0x00080000, // Schannel only
```

Comment:
For _most_ names above we try to fully type out `CERTIFICATE`. I think we should do so here as well.